.circleci/config.yml

```diff
@@ -3,12 +3,33 @@ version: 2.1
 orbs:
   hmpps: ministryofjustice/hmpps@3.1
 
+executors:
+  validator:
+    docker:
+      - image: cimg/openjdk:16.0
+      - image: localstack/localstack:0.12.9.1
+        environment:
+          - SERVICES=sqs,sns
+          - ES_PORT_EXTERNAL=4571
+          - DEBUG=${DEBUG- }
+          - DATA_DIR=/tmp/localstack/data
+          - DOCKER_HOST=unix:///var/run/docker.sock
+          - LAMBDA_EXECUTOR=${LAMBDA_EXECUTOR- }
+          - AWS_EXECUTION_ENV=True
+          - DEFAULT_REGION=eu-west-2
+          - TMPDIR=/private
+      - image: circleci/postgres:13.2
+        environment:
+          - POSTGRES_PASSWORD=admin_password
+          - POSTGRES_USER=admin
+          - POSTGRES_DB=court_register_db
+    environment:
+      _JAVA_OPTIONS: -Xmx256m -XX:ParallelGCThreads=2 -XX:ConcGCThreads=2 -Djava.util.concurrent.ForkJoinPool.common.parallelism=2 -Dorg.gradle.daemon=false -Dkotlin.compiler.execution.strategy=in-process -Dorg.gradle.workers.max=1
+    working_directory: ~/app
+
 jobs:
   validate:
-    executor:
-      name: hmpps/localstack
```

Comment:
A shame we had to go back to defining the exectuor but the orb doesn't support multiple supporting images.

Reply:
turns out is was very simple to add stuff to the orb, maybe if another project requires it then we should add it to the orb as well